packages/components/src/progress_bar/index.module.scss

```diff
@@ -0,0 +1,40 @@
+@import "nebenan-ui-kit";
+
+.root {
+  height: 20px;
+
+  overflow: hidden;
+  position: relative;
+
+  border-radius: $border-radius-m;
+  background: $color-gray-05;
+
+  color: $color-gray-09;
+  font-size: $font-size-s;
+  line-height: 20px;
+  text-align: right;
+}
+
+.progressState {
+  height: 100%;
+  position: absolute;
+  background: $color-green-02;
+  top: 0;
+  left: 0;
+
+  transition: width $transition;
+
+  em {
```

Comment:
Would prefer adding a class to the `em` element - won't be a need to nest it then